ui/package.json

```diff
@@ -8,44 +8,46 @@
   },
   "author": "Codeloop",
   "license": "ISC",
+  "private": true,
   "devDependencies": {
-    "babel-core": "6.26.x",
-    "babel-loader": "7.1.x",
-    "babel-plugin-transform-react-jsx": "6.24.x",
-    "babel-preset-env": "1.7.x",
-    "babel-preset-stage-2": "6.24.x",
-    "copy-webpack-plugin": "5.0.x",
-    "cross-env": "5.2.x",
-    "css-loader": "3.0.x",
     "eslint": "5.16.x",
     "eslint-config-airbnb": "17.1.x",
     "eslint-loader": "2.2.x",
     "eslint-plugin-import": "2.18.x",
     "eslint-plugin-jsx-a11y": "6.2.x",
-    "eslint-plugin-react": "7.14.x",
-    "file-loader": "4.0.x",
-    "html-webpack-plugin": "3.2.x",
-    "mini-css-extract-plugin": "0.8.x",
-    "node-sass": "4.12.x",
-    "postcss-loader": "3.0.x",
-    "@primer/css": "^12.4.x",
-    "sass-loader": "7.1.x",
-    "style-loader": "0.23.x",
-    "stylelint": "10.1.x",
-    "stylelint-config-primer": "7.0.x",
-    "stylelint-webpack-plugin": "0.10.x",
-    "webpack": "4.35.x",
-    "webpack-cli": "3.3.x",
-    "webpack-dev-server": "3.7.x"
+    "eslint-plugin-react": "7.14.x"
   },
   "dependencies": {
-    "@hyperapp/logger": "0.5.x",
-    "@hyperapp/router": "0.7.x",
-    "autoprefixer": "9.6.x",
-    "bootstrap": "4.3.x",
-    "hyperapp": "1.2.x",
-    "jquery": "3.4.x",
-    "popper.js": "1.15.x",
-    "whatwg-fetch": "3.0.x"
+    "bootstrap": "^4.3.1",
```

Comment:
- [ ] Please use `4.3.x` or exact version
- [ ] Change other as well

Reply:
Changed all except `react-bootstrap` to something like `9.9.x`.